wirepas_mesh_messaging/received_data.py

```diff
@@ -50,10 +51,11 @@ def __init__(
         event_id=None,
         hop_count=0,
         network_address=None,
+        time_ms_epoch=None,
         **kwargs
     ):
         super(ReceivedDataEvent, self).__init__(
-            gw_id, sink_id, event_id=event_id, **kwargs
+            gw_id, sink_id, event_id=event_id, time_ms_epoch=time_ms_epoch, **kwargs
```

Comment:
do we need to overload the constructor with time_ms_epoch? We should be able to pass in the time_ms_epoch parameter without having to add this parameter in the super constructor call.